BookPlayer/Player/PlayerViewController.swift

```diff
@@ -142,6 +139,48 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
         self.setNeedsStatusBarAppearanceUpdate()
     }
 
+    func updateToolbar(_ showTimerLabel: Bool = false, animated: Bool = false) {
+        guard var items = self.bottomToolbar.items else {
+            return
+        }
+
+        items = items.filter({ $0.tag > 0 })
```

Comment:
Since we already have the `IBOutlets` for all the buttons (except for the 'more' button), we could have:
```
var items = [
  self.speedButton,
  self.sleepButton
]

if currentBook.hasChapters {
  items.append(self.chaptersButton)
}

if #available(iOS 11, *) {
 .
 .
  items.append(avRoutePickerBarButtonItem)
}
.
.
```
This way we avoid relying on tags and it looks less convoluted 🤔, since we won't be deconstructing and array, but building the array

Reply:
with that approach we would have to add the 'more' button outlet though

Reply:
Feels better that way. Less trying to bend things into the right shape, more building the right structure from ground up.